.github/workflows/ci.yml

```diff
@@ -139,10 +139,12 @@ jobs:
         cd ui
         npm i
         npm run ci-test-integration
+        codecov
         cd ..
         cd provider-ui
         npm i
         npm test
+        codecov
```

Comment:
Remove this line and implement the **codecov** command with a GitHub Action in other PR.

```suggestion
        
```